hatchery/nextflow.go

```diff
@@ -66,18 +68,25 @@ func createNextflowResources(userName string, nextflowGlobalConfig NextflowGloba
 	// TODO The VPC, subnets, route tables and squid instance do not have the
```

Comment:
Can we file tickets for these TODOs?

Reply:
Yes i'll create tickets, we probably won't get to them very soon since these are "good to have" items. lmk if you see anything that should be prioritized.

Could you check [this](https://github.com/uc-cdis/hatchery/blob/c4d589691eb8504c327093a6681ce3a0143c5eed/hatchery/nextflow.go#L73) and lmk if we need to add more tags, so i can list them? if there is nothing to add, i'll remove the TODO

Reply:
We should at least have Environment, and then in some commons we have used `gen3username` but I'm not sure if we want to continue doing that. 

In the meantime the environment should be a tag so we can filter on which environment the resources are for.

Reply:
ticket MIDRC-746